index.js

```diff
@@ -210,9 +244,12 @@ class Tab extends EventEmitter {
     }
 
     setPosition (newPosition) {
+        if (newPosition === 0 || newPosition === undefined) {
+            return null;
+        }
         let tabContainer = this.tabGroup.tabContainer;
         let tabs = tabContainer.children;
-        let oldPosition = this.getPosition();
+        let oldPosition = this.getPosition() - 1;
```

Comment:
Can you explain this change please?

Reply:
This is to fix a bug due to the left position being 1 that I somehow didn't catch last PR. Sorry. :disappointed: